pkgs/by-name/zs/zsh-wd/package.nix

```
@@ -0,0 +1,38 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "wd";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "mfaerevaag";
+    repo = "wd";
+    rev = "v${version}";
+    sha256 = "sha256-u3n+JzInv8VajWAFXECuEOWuQurX8iWklwV2LG4Tj24=";
+  };
+
+  strictDeps = true;
+  dontBuild = true;
+
+  installPhase = ''
+    install -Dm555 wd.plugin.zsh $out/share/wd/wd.plugin.zsh
+    install -Dm555 wd.sh $out/share/wd/wd.sh
+    install -Dm555 _wd.sh $out/share/wd/_wd.sh
+    install -Dm444 wd.1 $out/share/wd/man/man1/wd.1
```

```suggestion
    installManpagewd.1
```

(needing `installShellFiles`)

---

Updated, this is so helpful! Also switched the installed script permissions to `755` and am installing shell completions with `installShellCompletion`.